### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by amantalwar04**
Updated. Please review.

### CODE REVIEW

**Review by shamilton1 — CHANGES_REQUESTED** [review]
Aman, I'd change line 4

File: README.md [hunk 1]
```diff
@@ -1,3 +1,4 @@
 # hello-world
 test repository
 ## Test Heading
+Updated in develop branch.
```
Comment (amantalwar04) [inline README.md]: @shamilton1  I think the updates are good. Can you please approve?

Comment (shamilton1) [reply README.md]: Change this line to add xyz parameters

**Comment by amantalwar04** [comment]
Updated the file as per @shamilton1 comments.

**Review by shamilton1 — APPROVED** [review]

**Comment by shamilton1** [comment]
Thanks

**Pull request closed without merging**